### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'iteritems'

### ISSUE THREAD

**Issue by RBlaine95** [kepoorhampond/irs#20]
I keep getting this error when trying to download a spotify playlist.

![information](https://cloud.githubusercontent.com/assets/4052340/25314445/b26475d4-284c-11e7-96c7-533551eb4acd.png)

```
irs -u "spotify" -p "Coffee Table Jazz"
Traceback (most recent call last):
  File "/usr/bin/irs", line 11, in <module>
    load_entry_point('irs==6.3.4', 'console_scripts', 'irs')()
  File "/usr/lib/python3.6/site-packages/irs/cli.py", line 59, in main
    ripper_args = dict((k, v) for k, v in ripper_args.iteritems() if v)
AttributeError: 'dict' object has no attribute 'iteritems'
```

**Comment by kepoorhampond** [comment]
Could you tell me which song it fails on?

**Comment by RBlaine95** [comment]
It fails to initialize. As soon as I execute the command, it bombs out with that error code.

**Comment by RBlaine95** [comment]
The command I'm executing is:
irs -u "spotify" -p "Coffee Table Jazz"

I tested it with a Spotify play list as well as a personal play list. Both gave the same error.

**Comment by kepoorhampond** [comment]
Fixed with PR #21

**Comment by RBlaine95** [comment]
Ah sweet, so, if I update irs on my computer using pip, the issue will be fixed?
